app.js

```diff
@@ -0,0 +1,17 @@
+const { getCategories } = require("./db/controllers/controllers.js");
+const express = require("express");
+const app = express();
+const {
+  handleCustomErrors,
+  handlePsqlErrors,
+  handleServerErrors,
+} = require("errors/error_handling.js");
+
+app.use(express.json());
+app.use(handleCustomErrors);
+app.use(handlePsqlErrors);
```

Comment:
Aside from the 500 error, we don't need the 400 and 404 error handlers just yet. We only want to implement these on an endpoint that needs them and after testing said endpoint :)